src/views/news/NewsExplore/NewsExplore.vue

```diff
@@ -18,7 +18,7 @@
         replace
         :isInSidebar="isInSidebar"
       />
-      <CalendarPanel :isInSidebar="isInSidebar" />
+      <CalendarPanel  v-if="isInSidebar" @page-update="monthFetch" :notableDates="calendarItems" :isInSidebar="isInSidebar"/>
```

Comment:
Do we need this v-if? Ideally it should still be accessible no matter which mode it's in